src/editor.c

```diff
@@ -1234,15 +1236,65 @@ editor_get_indent_prefs(GeanyEditor *editor)
 }
 
 
+static inline gint compare_line_indentation(ScintillaObject *sci, gint line1, gint line2)
```

Comment:
I always use `inline` on a function where I want it to inline the code, but not have to manually inline it in the source for readability reasons. I can remove the inline, though I think it adds a little semantic value for the reader and might hint the compiler to do the right thing, if it had any doubt.